src/uu/tr/src/tr.rs

```diff
@@ -59,53 +57,41 @@ pub fn uumain(args: impl uucore::Args) -> UResult<()> {
         .collect();
 
     if sets.is_empty() {
-        return Err(UUsageError::new(1, get_message("tr-error-missing-operand")));
+        return Err(UUsageError::new(
+            1,
+            translate!("tr-error-missing-operand", "util_name" => uucore::execution_phrase()),
```

Comment:
Is the "util_name" argument intentional?
```suggestion
            translate!("tr-error-missing-operand")
```